rootfs/api/models/app.py

```diff
@@ -800,11 +802,11 @@ def _build_env_vars(self, release):
             default_env['BUILDER_STORAGE'] = settings.APP_STORAGE
             default_env['DEIS_MINIO_SERVICE_HOST'] = settings.MINIO_HOST
             default_env['DEIS_MINIO_SERVICE_PORT'] = settings.MINIO_PORT
+            default_env['SOURCE_VERSION'] = release.build.sha
 
         # fetch application port and inject into ENV vars as needed
         port = release.get_port()
-        if port:
-            default_env['PORT'] = port
+        default_env['PORT'] = port
```

Comment:
this is the rebased commit from #1078, but essentially this is cleanup from dead code. `get_port` used to return a NoneType if the port didn't exist; now it just raises a DeisException or a numerical value so we don't need this check any more